src/js/views/admin/actions/actions_views.js

```diff
@@ -259,9 +260,19 @@ const AttachmentComponent = Droplist.extend({
   },
 });
 
+const AttachmentButton = View.extend({
```

Comment:
let's just change this to an icon like the status on a flow on a program page.

Reply:
so extend the Droplist and disable it on admin, but fire the click event on worklist/dashboard?

Reply:
no just an icon like this: https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/views/admin/program/workflows/flow-item.hbs#L6